README.md

```diff
@@ -0,0 +1,109 @@
+# Test Definitions
+A test suite works with and without [LAVA](http://lavasoftware.org/). The following two
+sets of automated tests are supported.
+
+- `automated/linux/`
+- `automated/android/`
+
+For each test case, both test script and corresponding test definition files provided in
+the same folder that named after test case name. Test scripts are self-contained and
+works independently. Test definition files in YAML format are provided for test run with
+local test-runner and within LAVA.
+
+## Installation
+Installing the latest development version:
+
+    git clone https://git.linaro.org/qa/test-definitions.git
+    cd ./test-definitions
+    . ./automated/bin/setenv.sh
+    pip install -r ${REPO_PATH}/automated/utils/requirements.txt
```

Comment:
Side note, does this repo support python3?

Reply:
test-runner supports both python2 and python3.